bin/install

```diff
@@ -237,8 +237,12 @@ download_and_verify_checksums() {
     fi
 
     (
-      if [ -z "${GNUPGHOME:-}" ] && [ -d "${ASDF_DIR:-$HOME/.asdf}/keyrings/nodejs" ]; then
-        export GNUPGHOME="${ASDF_DIR:-$HOME/.asdf}/keyrings/nodejs"
+      if [ -z "${GNUPGHOME:-}" ]; then
+        if [ -d "${ASDF_DIR:-$HOME/.asdf}/keyrings/nodejs" ]; then
+          export GNUPGHOME="${ASDF_DIR:-$HOME/.asdf}/keyrings/nodejs"
+        elif [ -d "${ASDF_DATA_DIR:-$HOME/.asdf}/keyrings/nodejs" ]; then
+          export GNUPGHOME="${ASDF_DATA_DIR:-$HOME/.asdf}/keyrings/nodejs"
+        fi
```

Comment:
I think it's more idiomatic to put the directories candidates into a loop so we can add more candidates later and get a more explicit priority, like so:
```suggestion
        for dir in "$ASDF_DIR" "$HOME/.asdf" "$ASDF_DATA_DIR"; do
          if [ -d "$dir/keyrings/nodejs" ]; then
            export GNUPGHOME="$dir/keyrings/nodejs"
          fi
        done
```

Reply:
Also, I would add a ASDF_NODEJS custom variable so it can be even more flexible

```suggestion
        for dir in "$ASDF_NODEJS_KEYRING_DIR" "$ASDF_DIR" "$HOME/.asdf" "$ASDF_DATA_DIR"; do
          if [ -n "$dir" ] && [ -d "$dir/keyrings/nodejs" ]; then
            export GNUPGHOME="$dir/keyrings/nodejs"
          fi
        done
```

Reply:
If you go with the ASDF_NODEJS_KEYRING_DIR idea, we should also add some documentation in the README explaining the functionality